hazelcast/src/main/java/com/hazelcast/util/ThreadUtil.java

```diff
@@ -54,4 +56,30 @@ public static void setThreadId(long threadId) {
     public static void removeThreadId() {
         THREAD_LOCAL.remove();
     }
+
+
+    /**
+     * Gets the threadname prefix.
+     *
+     * @param hzName the name of the hazelcast instance
+     * @param name   the basic name of the thread
+     * @return the created threadname prefix.
+     * @throws java.lang.NullPointerException if name is null.
+     */
+    public static String getThreadNamePrefix(String hzName, String name) {
```

Comment:
It was called `..Prefix` before but actually this is not a prefix, it's the thread name itself. So method name should be something like `getHazelcastInstanceThreadName(...)` or `createHazelcastInstanceThreadName(...)`...